react/src/utils/api.ts

```diff
@@ -68,7 +68,15 @@ export const getContacts = async (
       contactList.state = result.data.state;
       contactList.district = result.data.district;
       if (contactList.generalizedLocationID() !== '-') {
-        OneSignal.sendTag('districtID', contactList.generalizedLocationID());
+        const locationId = contactList.generalizedLocationID();
+        OneSignal.sendTag('districtID', locationId);
```

Comment:
[nitpick] should we use `districtId` in both places?

Reply:
👍